mailjet.cron.php

```diff
@@ -295,6 +295,10 @@ function findCode($code)
 					$content = str_replace('{'.$key.'}', $value, $content);
 				}
 
+                $content = str_replace('{shop_name}', $shop_name, $content);
+				$content = str_replace('{shop_url}', $shop_url, $content);
+				$content = str_replace('{shop_logo}', $shop_logo, $content);
+
```

Comment:
Hi Thomas :)
This one is fixed and the PR is updated.
The other one 
https://github.com/PrestaShop/mailjet/pull/24/files#r44911737  is OK for 
us - Thank you!

Regards!

On 11/16/2015 13:12 PM, Thomas N wrote:
>
> In mailjet.cron.php 
> <https://github.com/PrestaShop/mailjet/pull/24#discussion_r44911607>:
>
> > @@ -295,6 +295,10 @@ function findCode($code)
> >  					$content = str_replace('{'.$key.'}', $value, $content);
> >  				}
> >
> > +                $content = str_replace('{shop_name}', $shop_name, $content);
> > +				$content = str_replace('{shop_url}', $shop_url, $content);
> > +				$content = str_replace('{shop_logo}', $shop_logo, $content);
> > +
>
> You can make it quicker to execute:
>
> $tags_search = array('{shop_name}', '{shop_url}', '{shop_logo}');
> $tags_replace = array($shop_name, $shop_url, $shop_logo);
> $seltemplate = str_replace($tags_search, $tags_replace, $seltemplate);
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/PrestaShop/mailjet/pull/24/files#r44911607>.
>